app/scripts/lib/router.js

```diff
@@ -228,22 +227,6 @@ function (
       return viewOptions;
     },
 
-    _checkForRefresh: function (currentView) {
-      var refreshMetrics = this.storage.get('last_page_loaded');
-      var screenName = currentView.getScreenName();
-
-      if (refreshMetrics && refreshMetrics.view === screenName && this.metrics) {
```

Comment:
Yup, metrics is always created now. The backend drops metrics on the ground if not being saved for the user.